SysReqTableView.cpp

```diff
@@ -178,28 +178,28 @@ QVariant SysReqTableModel::data(const QModelIndex& index, int role) const {
       switch (index.row()) {
       case 0:
 #if ZEUS_ARCH_X86 || ZEUS_ARCH_X86_64
-        return QStringLiteral("x86_64");
+        return tr("x86_64");
 #else
         return {};
 #endif
       case 1:
-        return QStringLiteral("1.5 GHz");
+        return tr("1.5 GHz");
       case 2:
-        return QStringLiteral("3 GiB");
+        return tr("3 GiB");
       case 3:
-        return QStringLiteral("5 GB (MP1)");
+        return tr("5 GB (MP1)");
       case 4:
 #ifdef __APPLE__
-        return QStringLiteral("macOS 10.9");
+        return tr("macOS 10.9");
 #elif defined(_WIN32)
-        return QStringLiteral("Windows 7 SP1");
+        return tr("Windows 7 SP1");
 #elif defined(__linux__)
-        return QStringLiteral("Linux");
+        return tr("Linux");
 #else
         return {};
 #endif
       case 5:
-        return QStringLiteral("Blender 2.78");
+        return tr("Blender 2.78");
```

Comment:
While we're at it, let's go ahead and bump this to 2.80.

Reply:
Done! I've separated it into its own commit, since more than this string needed to be updated.

Reply:
Alright, thanks!